crates/starfish/core/src/block_header.rs

```diff
@@ -90,15 +90,12 @@ pub struct BlockHeaderV1 {
     epoch: Epoch,
     round: Round,
     author: AuthorityIndex,
-    // TODO: during verification ensure that timestamp_ms >= ancestors.timestamp
```

Comment:
removed because this is done in the `block_manager` verification step before the `DagState` write